db/schema.rb

```diff
@@ -19,6 +19,8 @@
     t.string   "icon_url"
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
+    t.string   "rating"
```

Comment:
Is there a way to amend that one commit to fix this and keep the child commits same?